.storybook/head.html

```diff
@@ -0,0 +1,15 @@
+<link href='https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css' rel='stylesheet' media="screen, print">
+<link href='https://cdnjs.cloudflare.com/ajax/libs/patternfly/3.10.0/css/patternfly.min.css' rel='stylesheet' media="screen, print">
+<link href='https://cdnjs.cloudflare.com/ajax/libs/patternfly/3.10.0/css/patternfly-additions.min.css' rel='stylesheet' media="screen, print">
+<script src='https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.4/jquery.min.js'></script>
```

Comment:
updated to a current jquery version.

I did try to use npm jquery package, however, since bootstrap is not in npm i was unable, as jquery needs to load prior to bootstrap.
the way that bootstrap works is by adding the bundle inside the body vs the head section.